portfolio/src/main/webapp/script.js

```diff
@@ -12,7 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
+// checks if text area is empty
```

Comment:
Better to be precise with these comments:

`// Checks if text area is empty, and if not, triggers an alert to the page`

Otherwise, a reader of this comment may call `emptyBox` and then call `alert` when `emptyBox` returns false, doubling up on the alerts.

Basically the idea i'm trying to convey is make sure any non-trivial side-effects of calling a function are documented in the comments.

Reply:
got it